pkg/ui/workspaces/db-console/src/views/databases/databaseDetailsPage/redux.ts

```diff
@@ -141,51 +133,48 @@ export const mapStateToProps = createSelector(
         databaseDetails[database]?.data?.results.tables_resp.tables,
         table => {
           const tableId = generateTableID(database, table);
-
           const details = tableDetails[tableId];
-          const stats = tableStats[tableId];
 
-          const roles = normalizeRoles(_.map(details?.data?.grants, "user"));
+          const roles = normalizeRoles(
+            _.map(details?.data?.results.grants_resp.grants, "user"),
+          );
           const grants = normalizePrivileges(
-            _.flatMap(details?.data?.grants, "privileges"),
+            _.flatMap(details?.data?.results.grants_resp.grants, "privileges"),
           );
-          const nodes = stats?.data?.node_ids || [];
+          const nodes = details?.data?.results.stats.ranges_data.node_ids || [];
           const numIndexes = _.uniq(
-            _.map(details?.data?.indexes, index => index.name),
+            details?.data?.results.schema_details.indexes,
           ).length;
           return {
             name: table,
+            loading: !!details?.inFlight,
```

Comment:
Is there a ticket to change this mapStateFromProps with some selectors on fields to a regular function rather than a selector?

Reply:
I don't believe so.